File: demo-sockets/demo-socket-server/src/main/java/com/labcompi1/message/SimpleMessage.java
package com.labcompi1.message;

import java.io.Serializable;

public class SimpleMessage implements Serializable {

    private String message;

    private static final long serialVersionUID = 6529685098267757690L;

    public SimpleMessage() {
    }

    public SimpleMessage(String message) {
        this.message = message;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    @Override
    public String toString() {
        return "SimpleMessage{" +
                "message='" + message + '\'' +
                '}';
    }
}

File: demo-sockets/demo-socket-client/app/src/main/java/com/labcompi1/message/SimpleMessage.kt
package com.labcompi1.message

class SimpleMessage(val message: String) : java.io.Serializable {

    companion object {
        private const val serialVersionUID = 6529685098267757690L
    }

    override fun toString(): String {
        return "SimpleMessage(message='$message')"
    }
}
